package.json

```diff
@@ -56,6 +58,7 @@
     "cross-env": "^5.2.0",
     "cucumber": "^5.1.0",
     "cucumber-html-reporter": "^4.0.4",
+    "download": "^7.1.0",
```

Comment:
What does `download` do? I can't see it used anywhere.

Reply:
Left in by mistake, now removed.